vnext/Microsoft.ReactNative/ReactPropertyBag.idl

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.ReactNative {
+
+  // The delegate is used to create property value on-demand.
+  [webhosthidden]
+  delegate Object ReactCreatePropertyValue();
+
+  // Namespace for the property name.
+  // Use ReactPropertyNameTable.GetNamespace to get atomic property namespace for a string.
+  [webhosthidden]
+  interface IReactPropertyNamespace {
+    // Get String name representation of the property namespace.
+    String NamespaceName { get; };
+  }
+
+  // Name of a property.
+  // Use ReactPropertyNameTable.GetName to get atomic property name for a string.
+  [webhosthidden]
+  interface IReactPropertyName {
+    // The local property name String in context of the property namespace.
+    String LocalName { get; };
+
+    // The namespace the property name is defined in.
+    IReactPropertyNamespace Namespace { get; };
```

Comment:
>Namespace [](start = 28, length = 9)

Are namespaces shared between property bags?

Reply:
Yes, the namespaces and names are global. They can be shared between multiple property bags. E.g. someone can create a static global PropertyName and then use it in any PropertyBag.